tools/cldr-apps/js/src/views/CldrError.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div style="background: #ffffff; padding: 30px">
+    <a-card hoverable :title="subtypeString">
+      <template #extra
+        ><a-tag :title="status.subtype" :color="status.type.toLowerCase()">{{
+          status.type
+        }}</a-tag></template
+      >
+      <p>
+        {{ status.message }}
+      </p>
+      <template v-if="status.subtypeUrl" #actions
+        ><a class="warningReference" :href="status.subtypeUrl"
+          >How to Fix…</a
+        ></template
+      >
+    </a-card>
+  </div>
+</template>
+
+<script>
+// import * as cldrLoad from "../esm/cldrLoad.mjs";
+// import * as cldrStatus from "../esm/cldrStatus.mjs";
+export default {
+  computed: {
+    myClass() {
+      return `cldrError tr_${this.status.type} `;
+    },
+    subtypeString() {
+      return this.status.subtype
+        .split(/(?=[A-Z])/)
+        .map((s) => s)
+        .join(" ");
+    },
+  },
+  props: {
+    status: {
+      type: Object,
+      // any additional classes
+      default: "",
```

Comment:
should message, subtype and subtypeUrl be declared here? It looks like they're filled in from json from the back end. If that fails for any reason, the front end will throw cryptic errors like undefined subtype... I know we already have a lot of js code that makes this kind of implicit assumption. Still, objects with clearly defined and documented properties would be more ideal. I think status corresponds to back-end CheckCLDR.CheckStatus, and subtype corresponds to CheckCLDR.CheckStatus.Subtype. Comments could help...

Reply:
I don't think it will type check here.